### PR TITLE
Fix: warning on checkout page

### PR DESCRIPTION
**Pull request by Splagov**
- fixed issue when warning on checkout page apper if zipcode doesn't match selected state.

### Related issue(s)
#2611 

### Steps to reproduce & screenshots/GIFs
1. Go to checkout 'Shopping Cart' page
2. In shipping details fill shipping address, with mistake, where zip code doesn't match selected state.
3. Refresh page.
4. See warning above cart info.

Before: (wrong state selected)
image

After: (wrong state selected)
image

### Checklist

- [ ] unit tests

- [ ] `changelog.txt` entry added

### CODE REVIEW

File: classes/class-wc-connect-taxjar-integration.php [hunk 1]
```diff
@@ -913,6 +913,8 @@ public function append_base_address_to_customer_taxable_address( $address ) {
 	 * @return object
 	 */
 	public function maybe_override_taxjar_tax( $taxjar_resp_tax, $body ) {
+		if( ! isset($taxjar_resp_tax) ) return;
```
Comment (bartech) [inline classes/class-wc-connect-taxjar-integration.php]: WordPress Coding standards: Inline control structures are not allowed.
```suggestion
		if ( ! isset( $taxjar_resp_tax ) ) {
			return;
		}
```

**Review by bartech — CHANGES_REQUESTED** [review]
I run the code on my local and looks good. 
Please only address code formatting.

**Review by bartech — APPROVED** [review]

**Comment by Splagov** [comment]
how I can run JS Eslint and Test and PHP Syntax Check  ?

**Review by bartech — APPROVED** [review]

**Comment by harriswong** [comment]
Added missing changelog and readme entry in this commit: https://github.com/Automattic/woocommerce-services/commit/087b5831d9d9532cb31e456869536ce61ab96655

**Pull request merged**